frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/strings/renderable-field.ts

```diff
@@ -4,20 +4,29 @@ import {
   VMSettingsField,
   RenderableFieldResolver,
   VMWareProviderField,
+  OvirtProviderField,
 } from '../types';
 import { ProvisionSource } from '../../../constants/vm/provision-source';
 
 export const titleResolver: RenderableFieldResolver = {
```

Comment:
I struggled a bit what should be the strings here? Are we using upstream or downstream name? And it seems a bit too much repeating Red Hat Virtualization everywhere.

@matthewcarleton

Reply:
I’ve struggled with that too. I wonder can we get away with in the drop down having Red Hat Virtualization (RHV) and then just use RHV from there on out? We would want to ensure we make it clear to screen readers.

Reply:
nice, looks much better,

![aa](https://user-images.githubusercontent.com/3648838/79210111-a0456600-7e44-11ea-9582-e3707f573cbf.png)

Reply:
I left the long version int the statuses in the VMs list as there is no repetition

Reply:
> We would want to ensure we make it clear to screen readers.

Isn't it enough to be the same what normal user sees?  Where would we want to add it? To the label? - which is quite hard as we are using PF `FormGroup` for that

Reply:
ya, you might be right. As long as they make the connection between the provider selection and then the label of the dropdown.